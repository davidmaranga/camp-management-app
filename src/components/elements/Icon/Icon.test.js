import React from 'react';
import { shallow } from 'enzyme';
import Icon from './index';

describe('RENDERING - Icon', () => {
  it('should render with default props', () => {
    const component = shallow(<Icon icon="test" />);
    expect(component).toMatchSnapshot();
  });
});
