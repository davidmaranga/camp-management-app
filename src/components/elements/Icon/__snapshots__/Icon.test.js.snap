// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RENDERING - Icon should render with default props 1`] = `
<i
  className="Icon"
>
  test
</i>
`;
